deps/rabbitmq_prometheus/src/collectors/prometheus_process_colletor.erl

```diff
@@ -0,0 +1,64 @@
+-module(prometheus_process_colletor).
```

Comment:
```suggestion
-module(prometheus_process_collector).
```

kudos to @ViliusS for catching this 🙂 https://github.com/rabbitmq/rabbitmq-server/issues/4539#issuecomment-1182556950

Reply:
good catch, thanks